src/aims_ui/api_interaction.py

```diff
@@ -1,33 +1,40 @@
+import os
+import json
 import requests
 from . import app
+from flask import render_template
+from .models.get_endpoints import get_endpoints
 
+def get_params(all_user_input):
+  """Return a list of aparameters formatted for API header, from class list of inputs"""
+  params = 'verbose=True&&'
```

Comment:
Should this be a double-&? If this is meant to be an HTTP parameter string, it only needs a single &. The same is true a few lines down.

Reply:
It should be a single and, have adjusted